src/main/resources/lib/nextjs/postprocessing.es6

```diff
@@ -1,18 +1,23 @@
 /** Replace URL refs in both HTML, JS and JSON sources from pointing to frontend-urls to making them sub-urls below the extFrontendProxy service */
-export const getBodyWithReplacedUrls = (req, body, proxyUrlWithSlash,) => {
-    // Replace CSS urls in the format: <some-property>: url('</some/url>')>
-    // Do this for JS and HTML files as well to support:
-    // import "../styles.css" in JS
-    // <style>.style {}</style> in HTML
-    // (Quotes are optional because next doesn't use them in production mode)
-    const cssUrlPattern = new RegExp(`url\(\s*(['"\`])?\/([^'"\`]*)['"\`]?\s*\)`, "gm");
+export const getBodyWithReplacedUrls = (req, body, proxyUrlWithSlash, isCss) => {
+    // double slashes is the right way of escaping bracket!
+    const cssUrlPattern = new RegExp(`url\\([ \r\n\t]*(['"\`])?\/([^'"\`]*)['"\`]?[ \r\n\t]*\\)`, "g");
 
     // Replace local absolute root URLs (e.g. "/_next/..., "/api/... etc)
     const nextApiPattern = new RegExp(`(['"\`])([^'"\` \n\r\t]*\/)((?:_next(?!\/image?)\/|api\/)[^'"\` \n\r\t]*)['"\`]`, "g");
 
-    return body
-        .replace(nextApiPattern, `$1${proxyUrlWithSlash}$3$1`)
-        .replace(cssUrlPattern, `url($1${proxyUrlWithSlash}$2$1)`);
+    // Don't do next static urls replacement in css
+    const result = isCss ?
+        body :
+        body.replace(nextApiPattern, `$1${proxyUrlWithSlash}$3$1`);
```

Comment:
return isCss ?
        body.replace(cssUrlPattern, `url($1${proxyUrlWithSlash}$2$1)`) : 
        body.replace(nextApiPattern, `$1${proxyUrlWithSlash}$3$1`);
        
Would this be possible? Or do we need to double replace in some cases?

Reply:
Yes, we need both regexps in case it is not css